scipy/stats/stats.py

```diff
@@ -3692,6 +3692,103 @@ def f_oneway(*args, axis=0):
     return F_onewayResult(f, prob)
 
 
+AlexanderGovernResult = namedtuple("AlexanderGovernResult", ("statistic",
+                                                             "pvalue"))
+
+
+def AlexanderGovern(*args):
+    """
+    Performs the Alexander Govern test.
+
+    The Alexander-Govern approximation tests the equality of k independent
+    means in the face of heterogeneity of variance. The test is applied to
+    samples from two or more groups, possibly with differing sizes.
+
+    Parameters
+    ----------
+    sample1, sample2, ... : array_like
+        The sample measurements for each group.  There must be at least
+        two arguments.
+
+    Returns
+    -------
+    statistic : float
+        The computed A statistic of the test.
+    pvalue : float
+        The associated p-value from the chi-squared distribution.
+
+    References
+        ----------
+        ... [1] Alexander, Ralph A., and Diane M. Govern. “A New and Simpler
+                Approximation for ANOVA under Variance Heterogeneity.” Journal
+                of Educational Statistics, vol. 19, no. 2, 1994, pp. 91–101.
+                JSTOR, www.jstor.org/stable/1165140. Accessed 12 Sept. 2020.
+    Examples
+        --------
```

Comment:
Add some commentary explaining what's going on. If this data is from the original paper, say so, otherwise identify the source. As an example, see gh-12531 docstring.